examples/src/examples/misc/gizmos.mjs

```diff
@@ -695,6 +778,36 @@ class GizmosExample {
     static WEBGPU_ENABLED = false;
     static controls = controls;
     static example = example;
+    static FILES = {
```

Comment:
Not sure the example needs to have non-clipping version of the grid, the example is about gizmos. This makes it more complex outside of its scope.

Reply:
Alright ill revert those changes 👍